Practice Areas/Engineering/code_review_guidelines.md

```diff
@@ -12,12 +12,21 @@ Before sending your code to a VSP team for review, your code should meet all of
 ## Guidelines
 
 * **Review lifecycle**
-    * Before sending code to VSP team, have it reviewed by your team (see **First review** below).
-    * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to Leah and Patrick.
+    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).  
+      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
+    * After having your pull request reviewed by your team you can mark your pull request as ready for review.
+      * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to your lead or scrum master.
+      * Please **do not post in Slack** channels requesting code reviews unless it has been more then 24 hours (business days).
     * After sending your code for review, don't add additional changes! Code needs to be stable for a safe review.
-    * If non-trivial changes to your code are made *after approval by VSP team*, your change is subject to being reverted.
-
-* **First review**: your code should first be reviewed and approved by a member of your team. That first review should cover:
+    * If non-trivial changes to your code are made *after approval by VSP team*, please notify approvers so they can re-review your pull request.
+
+* **First review**: 
+  * Use a [Draft Pull Request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests)
+  * Add your team specific label(s)
+    * Additionally if your change requires:
+      * A database migration please use the `database-migration` label
+      * A dependency/package upgrade please use the `package-change` label
```

Comment:
Does this label exist right now?

Reply:
This label doesn't exist in vets-api.  Can you explain the need for this label?

Reply:
Easier for reviewers to know at a glance when PRs contain database or package/dependency changes

Reply:
Created the labels for 
- https://github.com/department-of-veterans-affairs/gibct-data-service
- https://github.com/department-of-veterans-affairs/vets-api
- https://github.com/department-of-veterans-affairs/vets-website

Reply:
Hmm, I don't think we need labels for that though. If a PR contains database or package changes, call that out in the issue heading and/or PR description?

I'll leave this up to @annaswims to make a final call on whether or not we want this, but in general we have a preference _against_ adding extra process until there's a clear need for it.

Reply:
I agree with @gunsch. I think this kind of indication can be added in the PR description as needed.

In general, PR descriptions should be pretty thorough, so it seems like a good fit there.

Reply:
Which would be better?
```
  * Tag your PR with the following labels:
    * For *all code*, add your team-specific label(s)
    * (Optional) For *database migrations*, add the `database-migration` label
    * (Optional) For *dependency/package upgrades*, add the `package-change` label
```
or
```
  * Tag your PR with the following labels:
    * For *all code*, add your team-specific label(s)
```